src/vehicle.cpp

```diff
@@ -109,6 +109,15 @@ enum vehicle_controls {
  toggle_scoop
 };
 
+enum vehicle_controls_simple {
```

Comment:
No need to make a separate enum for all this stuff, you could (probably should) reuse the enum that already exists.
You could make a set of control options available for simple controls like this:
```
const auto simple_options = std::set<vehicle_controls>() { toggle_cruise_control, activate_horn, [rest of the stuff here], convert_vehicle };
```
And check if a given control option is available at simple controls with `simple_options.count( option ) > 0`.